### DISCUSSION TITLE
Skipping AC-PC alignment step

### DISCUSSION THREAD

**Discussion by chrissongwi** [PennLINC/qsiprep#575] [Q&A]
Hello. I am new to QSIprep. QSIprep seems to do an AC-PC alignment by default. Is there any way I can skip this step since I want to use the existing AC-PC aligned T1? Thank you.

**Comment by mattcieslak** [comment]
In the current versions there is no way to skip it, but in the upcoming months you will be able to specify an anatomical reference (which would be your previously ac-pc aligned T1w).
